plutus-use-cases/test/Spec/DynamicLogic/RegistryModel.hs

```diff
@@ -0,0 +1,263 @@
+{-# LANGUAGE FlexibleInstances  #-}
```

Comment:
Also, are these tests actually hooked up?

Reply:
No. The models for the example contracts are also not hooked up to the test suite. What would be the best way of doing that?

Reply:
It's all just `tasty`, so mostly a matter of making a `tasty` `Test` and hooking it up to the one in the top-level `Spec.hs`.